pyaedt/application/Design.py

```diff
@@ -2105,7 +2106,10 @@ def insert_design(self, design_name=None):
 
         """
         if self.design_type == "Circuit Design" or self.design_type == "HFSS 3D Layout Design":
-            self.modeler.edb.close_edb()
+            try:
+                self.modeler.edb.close_edb()
+            except:
```

Comment:
It's a bit of an antipattern to have naked `except:`.  Let's not do this in the future.

Reply:
this is needed to bypass the aedt_exception_handler. Anyway I've replaced with if statement